### PR TITLE
ath79: add tl-wr1043nd-v4 support

### PR DESCRIPTION
**Pull request by lucize**
tplink-safeloader: resize kernel partition
kernel 4.14 is bigger and it will not fit anymore, resize the partition, we have plenty of space left

everything is working

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

### CODE REVIEW

**Comment by lucize** [comment]
based on #1043

**Comment by hackpascal** [comment]
Hi, please be noticed that I've re-pushed the commit and the file qca956x.dtsi has been changed.

**Comment by lucize** [comment]
I'll drop the file so it will have no conflicts after yours PR gets merged, the dtsi is also working for ubiquiti ap ac lite/lr

File: target/linux/ath79/dts/qca9563_tl-wr1043n.dtsi [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <25000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+				partition@0 {
```
Comment (karlp) [inline target/linux/ath79/dts/qca9563_tl-wr1043n.dtsi]: indentation is wrong here.

File: target/linux/ath79/dts/qca9563_tl-wr1043n.dtsi [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+				label = "u-boot";
+				reg = <0x000000 0x020000>;
+				read-only;
+			};
+
+			partition@20000 {
```
Comment (karlp) [inline target/linux/ath79/dts/qca9563_tl-wr1043n.dtsi]: just fyi, you can also just do "firmware@20000" and skip the label properties.

**Comment by blogic** [comment]
merged, Thanks !

**Pull request closed without merging**